src/Sylius/Bundle/ApiBundle/DataTransformer/AddProductReviewInputDataTransformer.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\DataTransformer;
+
+use Sylius\Bundle\AdminApiBundle\Model\UserInterface;
+use Sylius\Bundle\ApiBundle\Command\AddProductReview;
+use Sylius\Bundle\ApiBundle\Context\UserContextInterface;
+use Sylius\Component\Core\Model\CustomerInterface;
+use Sylius\Component\Core\Model\ShopUserInterface;
+
+/** @experimental */
+final class AddProductReviewInputDataTransformer implements CommandDataTransformerInterface
```

Comment:
I wonder if we should do a more generic data transformer `LoggedInShopUserEmailAwareCommandDataTransformer` similar to `LoggedInShopUserIdAwareCommandDataTransformer`